packages/rpm-spec-language-server/package.yaml

```diff
@@ -0,0 +1,19 @@
+---
+name: rpm-spec-language-server
```

Comment:
The name of the package should in general be the same as the one of the binary, that is `rpm_lsp_server`. (Note that the name of the parent folder needs to match that as well.)